### PR TITLE
Update guest-access-PowerShell.md

### PR DESCRIPTION
**Pull request by cpoDesign**
Added command with how to install the PS module

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/guest-access-PowerShell.md [hunk 1]
```diff
@@ -37,18 +37,27 @@ You can also use PowerShell to allow or block a guest user based on their domain
   
 If you want to block guests in Teams and still want to allow them to access SharePoint sites, you can use Azure AD PowerShell cmdlets to disable the AllowGuestsToAccessGroups parameter on the Company object, assuming external sharing is turned on for SharePoint sites.
 
+> [!NOTE]
+> This guide is dependant on the following module
```
Comment (JohanFreelancer9) [inline Teams/guest-access-PowerShell.md:41]:
```suggestion
> This guide is dependant on the following module:
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@MikePlumleyMSFT v

**Comment by MikePlumleyMSFT** [comment]
This PR adds conflicting information. We'll schedule an update for this article to clarify which PS modules are needed.

**Pull request closed without merging**